### ISSUE TITLE
[CA1865] Fixer reformats code

### ISSUE THREAD

**Issue by MaceWindu** [dotnet/roslyn-analyzers#6930]
### Analyzer

**Diagnostic ID**: [CA1865](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1865-ca1867): `Use 'string.Method(char)' instead of 'string.Method(string)' for string with single char`

### Analyzer source

**NuGet Package**: [Microsoft.CodeAnalysis.NetAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.NetAnalyzers)

**Version**: 8.0.0-preview.23420.2

### Describe the bug

For multiline statement fixer removes newline after fixed method call if it follows closing bracket ')' directly or have comment on next line

### Steps To Reproduce

```cs
public class Class1
{
	public static void Test(string str)
	{
		// newline preserved due to comment below
		if (str.StartsWith(".", StringComparison.Ordinal)
			// space after ) - newline preserved
			|| str.EndsWith(".", StringComparison.Ordinal) 
			// don't trigger unwrap
			|| str.IndexOf(".", StringComparison.Ordinal) == 3
			// // newline removed
			|| 2 == str.IndexOf(".", StringComparison.Ordinal)
			|| str == "test")
		{
			return;
		}
	}
}


```

### Expected behavior

```cs
public class Class1
{
	public static void Test(string str)
	{
		// newline preserved due to comment below
		if (str.StartsWith('.')
			// space after ) - newline preserved
			|| str.EndsWith('.')
			// don't trigger unwrap
			|| str.IndexOf('.') == 3
			// // newline removed
			|| 2 == str.IndexOf('.')
			|| str == "test")
		{
			return;
		}
	}
}

```

### Actual behavior

```cs
public class Class1
{
	public static void Test(string str)
	{
		// newline preserved due to comment below
		if (str.StartsWith('.')
			// space after ) - newline preserved
			|| str.EndsWith('.')
			// don't trigger unwrap
			|| str.IndexOf('.') == 3
			// // newline removed
			|| 2 == str.IndexOf('.') || str == "test")
		{
			return;
		}
	}
}

```

**Comment by mpidash** [comment]
The fixer does not copy trivia from the node it replaces, resulting in loss of formatting.
I can prepare a PR for this.
